timur/lib/client/jsx/components/browser/browser.jsx

```diff
@@ -189,12 +189,12 @@ function useEditActions(setMode, browserState) {
 
   function postEdits() {
     setMode('submit');
-    sendRevisions(
+    invoke(sendRevisions(
```

Comment:
@coleshaw This is all you need to get local dev working agian.

Reply:
Nice! Thanks!

Reply:
I can't comment outside of the changes...but does `view` need to be passed from `selectOrShowTab` to `showTab`?

```
  function selectOrShowTab(view) {
    if (currentTabName) showTab(view);
    else selectDefaultTab(view);
  }

  function showTab(view) {
    requestTabDocuments(view.tabs[currentTabName]);
    setMode('browse');
  }
```

The current data isn't showing up correctly on page load... (note that adding the `view` parameter doesn't fix things, but does make them better).

Reply:
Okay, so the above comment about `view` may not be required ... but I'm not getting data loaded correctly somehow. I think there's something else going on in addition to the `invoke` call.

Reply:
:+1: rebuilt all my docker stuff and it works now. Thanks!